DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -140,6 +140,30 @@ public function process(ContainerBuilder $container)
                     $groups[$resolvedGroupName]['roles'] = $groupDefaults[$resolvedGroupName]['roles'];
                 }
             }
+        } elseif ($container->getParameter('sonata.admin.configuration.sort_admins')) {
+            $groups = $groupDefaults;
+
+            $label_or_admin = function ($e) {
+                return !empty($e['label']) ? $e['label'] : $e['admin'];
+            };
+
+            // sort the default group definition
+            ksort($groups);
+            array_walk(
```

Comment:
I tried to change the sorting now - there still is the array_walk with the call to a function but I moved that one outside. Better?